tutorcelery/patches/k8s-deployments

```diff
@@ -76,11 +76,38 @@ spec:
           image: {{CELERY_FLOWER_DOCKER_IMAGE}}
           ports:
             - containerPort: 5555
+              name: metrics
           env:
           - name: CELERY_BROKER_URL
             value: redis://{{ REDIS_USERNAME }}:{{ REDIS_PASSWORD }}@{{ REDIS_HOST }}:{{ REDIS_PORT }}/{{ OPENEDX_CELERY_REDIS_DB }}
           - name: FLOWER_PORT
             value: "5555"
           - name: FLOWER_BASIC_AUTH
             value: {{CELERY_FLOWER_BASIC_AUTH}}
+
+{% if CELERY_FLOWER_PROMETHEUS -%}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app.kubernetes.io/name: flower
+    release: prometheus
```

Comment:
Maybe we should add a note saying that the `release: prometheus` label is the default that Prometheus uses to select service monitors. This won't work for a Prometheus with a different ServiceMonitor label selector